conda-recipe/meta.yaml

```diff
@@ -67,9 +67,12 @@ outputs:
         - z5py
         - zarr 2.*
       run_constrained:
-        - mkl !=2024.1.0
+        - mkl <2024.1.0
```

Comment:
:(

Reply:
yeah, hope this is resolved soon (related: https://github.com/pytorch/pytorch/issues/129310)